video/quickstart.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2017, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// [START videointelligence_quickstart]
+// Imports the Google Cloud Video Intelligence library
+const Video = require('@google-cloud/videointelligence').v1beta1();
+
+// Instantiates a client
+const video = Video.videoIntelligenceServiceClient();
```

Comment:
Does project ID not need to be specified here? All the other quickstarts demonstrate passing in a `projectId` to the client.